groups/custom-elements.yml

```diff
@@ -1,3 +1,3 @@
 # https://html.spec.whatwg.org/multipage/custom-elements.html
 name: Custom elements
-parent: html
+parent: web-components
```

Comment:
I think it's weird that custom elements are not part of `elements`.

Reply:
We could allow groups to have multiple parents if that's required to match reality here. However, I would consider custom elements as a sibling group to built-in elements.

I'm not sure about the relationship between custom elements and web components. To start out, can we leave custom elements with HTML as the parent group, and use both custom elements and web components as groups for the features that belong in both?